integration-libs/opf/components/opf-checkout-payments/opf-checkout-payments.component.ts

```diff
@@ -33,12 +37,13 @@ export class OpfCheckoutPaymentsComponent implements OnInit, OnDestroy {
   activeConfiguratons$ = this.opfCheckoutService
     .getActiveConfigurationsState()
     .pipe(
-      tap((state) => {
-        if (state?.error) {
-          this.globalMessageService.add(
-            { key: 'opf.checkout.errors.loadActiveConfigurations' },
-            GlobalMessageType.MSG_TYPE_ERROR
-          );
+      tap((state: QueryState<ActiveConfiguration[] | undefined>) => {
+        if (state.error) {
+          this.displayError('loadActiveConfigurations');
+        } else {
+          if (!state.loading && !Boolean(state.data?.length)) {
```

Comment:
Why not to use else if instead of placing nesting if statement inside of else block?

Reply:
done, thanks